packages/visx-demo/src/sandboxes/visx-shape-line-radial/Example.tsx

```diff
@@ -92,32 +96,43 @@ const Example = ({ width, height, animate = true }: LineRadialProps) => {
         <LinearGradient from={green} to={blue} id="line-gradient" />
         <rect width={width} height={height} fill={background} rx={14} />
         <Group top={height / 2} left={width / 2}>
-          {/** Radial circles */}
-          {yScaleTicks.map((tick, i) => (
-            <circle
-              key={`radial-grid-${i}`}
-              r={yScale(tick)}
-              stroke={blue}
-              strokeWidth={1}
-              fill={blue}
-              fillOpacity={1 / (i + 1) - (1 / i) * 0.2}
-              strokeOpacity={0.2}
-            />
-          ))}
-          {/** Labels on top */}
-          {yScaleTicks.map((tick, i) => (
-            <text
-              key={`radial-grid-${i}`}
-              y={-(yScale(tick) ?? 0)}
-              dy="-.33em"
-              fontSize={8}
-              fill={blue}
-              textAnchor="middle"
-            >
-              {tick}
-            </text>
-          ))}
-
+          <GridAngle
+            scale={xScale}
+            outerRadius={height / 2 - padding}
+            stroke={green}
+            strokeWidth={1}
+            strokeOpacity={0.3}
+            strokeDasharray="5,2"
+            numTicks={20}
+          />
+          <GridRadial
+            scale={yScale}
+            numTicks={5}
+            stroke={blue}
+            strokeWidth={1}
+            fill={blue}
+            fillOpacity={0.1}
+            strokeOpacity={0.2}
+          />
+          <AxisLeft
+            top={-height / 2 + padding}
+            scale={reverseYScale}
+            numTicks={5}
+            tickStroke="none"
+            tickLabelProps={val => ({
+              fontSize: 8,
+              fill: blue,
+              fillOpacity: 1,
+              textAnchor: 'middle',
+              dx: '1em',
+              dy: '-0.5em',
+              stroke: bg,
+              strokeWidth: 0.5,
+              paintOrder: 'stroke',
+            })}
+            tickFormat={formatTicks as any}
```

Comment:
can we avoid this `any`? I think right now you have it returning a `number` whereas [it expects a `string`](https://github.com/airbnb/visx/blob/master/packages/visx-axis/src/types.ts#L16)